components/ActionSheet/docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Action Sheet
+
+<!-- badges -->
+
+Material design action sheets should be used as an overflow menu. An Action Sheet comes up from the bottom of
+the screen and displays actions a user can take.
+
+<img src="assets/actionSheetPortrait.gif" alt="An animation showing a Material Design Action Sheet." width="150"> <img src="assets/actionSheetLandscape.gif" alt="An animation showing a Material Design Action Sheet." width="324">
+
+<!-- design-and-api -->
+
+<!-- toc -->
+
+- - -
+
+## Overview
+
+`MDCActionSheetController` is a material design implemenation of UIAlertControllerStyleActionSheet.
+
+## Installation
+
+- [Typical installation](../../../docs/component-installation.md)
+
+## Usage
+
+- [Typical use](typical-use.md)
```

Comment:
Because this API mimics UIKit's so closely it may be helpful to the reader to have a comparison guide of the two APIs. Notably call out:

1. What's the same.
2. What's different.

Reply:
Done